lib/i18n/html_extractor/match.rb

```diff
@@ -16,7 +17,13 @@ def initialize(document)
         end
 
         def matches
```

Comment:
Rubocop complained when I tacked on `+ alt_matches(document)` at the end of the inner block.

I rewrote the concatenation to use `reduce(&:concat)`; open to suggestions if this could be clearer!